components/Header/Header.tsx

```diff
@@ -6,10 +6,12 @@ import useConnectWallet from "@/hooks/useConnectWallet";
 import Button from "../Button";
 import LoginButton from "../LoginButton";
 import Tooltip from "../ui/Tooltip";
+import usePrivyAddress from "@/hooks/usePrivyAddress";
 
 const Header = () => {
   const { push } = useRouter();
-  const { address, connectWallet } = useConnectWallet();
+  const { connectWallet } = useConnectWallet();
+  const address = usePrivyAddress();
```

Comment:
Please follow our existing standard for exporting variables from a hook.
```suggestion
  const {address} = usePrivyAddress();
```